securitycenter/notifications/delete_notification_config.go

```diff
@@ -0,0 +1,38 @@
+package notifications
+
+// [START scc_delete_notification_config]
+import (
+	"context"
+	"fmt"
+
+	securitycenter "cloud.google.com/go/securitycenter/apiv1p1beta1"
+	securitycenterpb "google.golang.org/genproto/googleapis/cloud/securitycenter/v1p1beta1"
+)
+
+func deleteNotificationConfig(orgId string, notificationConfigId string) error {
+	ctx := context.Background()
+	client, err := securitycenter.NewClient(ctx)
+
+	if err != nil {
+		return fmt.Errorf("securitycenter.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	// TODO orgId := "your-org-id"
+	// TODO notificationConfigId := "config-to-delete"
+
+	name := fmt.Sprintf("organizations/%s/notificationConfigs/%s", orgId, notificationConfigId)
+	req := &securitycenterpb.DeleteNotificationConfigRequest{
+		Name: name,
+	}
+
+	err = client.DeleteNotificationConfig(ctx, req)
+	if err != nil {
```

Comment:
Inline error declaration -- limits scope.

```suggestion
	if err = client.DeleteNotificationConfig(ctx, req); err != nil {
```

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#return-errors

Reply:
Done